src/tools/rustfmt/src/chains.rs

```diff
@@ -232,7 +232,7 @@ impl ChainItemKind {
                 let span = mk_sp(nested.span.hi(), field.span.hi());
                 (kind, span)
             }
-            ast::ExprKind::Await(ref nested) => {
+            ast::ExprKind::Await(ref nested, _) => {
```

Comment:
Changes to rustfmt are basically no-ops, just adding a new field to an ast variant. Sorry for the ping :)

Reply:
more spans/more precise spans are just about always beneficial for rustfmt so no complaints here :)